<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btn_Groups">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Groups</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Subjects">
         <property name="text">
          <string>Subject</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_TimeBlocks">
         <property name="text">
          <string>Time Blocks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Teachers">
         <property name="text">
          <string>Teachers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Rooms">
         <property name="text">
          <string>Rooms</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Constraints">
         <property name="text">
          <string>Constraints</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Checkout">
         <property name="text">
          <string>Checkout</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>6</number>
      </property>
      <widget class="QWidget" name="groupsPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="4,0,6">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_4" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line">
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_3" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <widget class="QLabel" name="label_9">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>160</y>
             <width>67</width>
             <height>17</height>
            </rect>
           </property>
           <property name="text">
            <string>preview</string>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="teachersPage">
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="4,0,0,6">
        <item row="0" column="0" colspan="2">
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Id</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="2" column="1">
              <widget class="QListWidget" name="listWidget_2">
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Subjects</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QWidget" name="widget_7" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="roomsPage">
       <layout class="QGridLayout" name="gridLayout" columnstretch="4,0,6">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>ID</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Features</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_3">
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item row="2" column="1">
              <widget class="QListWidget" name="listWidget">
               <property name="selectionMode">
                <enum>QAbstractItemView::MultiSelection</enum>
               </property>
               <property name="sortingEnabled">
                <bool>true</bool>
               </property>
               <item>
                <property name="text">
                 <string>computer</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>projector</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_2">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_5" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="subjectsPage">
       <layout class="QGridLayout" name="gridLayout_4" columnstretch="4,0,6">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QFormLayout" name="formLayout_4">
             <item row="0" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Room features</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Teacher subject</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
             <item row="1" column="1">
              <widget class="QListWidget" name="listWidget_3"/>
             </item>
             <item row="2" column="1">
              <widget class="QListWidget" name="listWidget_4"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line_5">
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_9" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="constraintsPage">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>60</y>
          <width>151</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>constaints</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="checkoutPage">
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>110</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>checkout</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="timeBlocksPage">
       <layout class="QGridLayout" name="gridLayout_5" columnstretch="4,0,6">
        <item row="0" column="0">
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Day</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Start hour</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>End hour</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QListWidget" name="listWidget_5">
               <item>
                <property name="text">
                 <string>Monday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Tuesday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Wednesday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Thursday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Friday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Saturday</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sunday</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_8"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_9"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_5">
             <property name="text">
              <string>Submit</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="Line" name="line_6">
          <property name="lineWidth">
           <number>3</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QWidget" name="widget_11" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
